content/en/docs/KRE/training/_index.md

```diff
@@ -36,3 +40,265 @@ Once deployed the generated KRT file, you can open an entrypoint to your local p
 ### Result example
 
 {{< imgproc kai-training-result-example Resize "1000x" />}}
+
+
+This step-by-step page will guide you creating a new Konstellation version for your project that will help you to
```

Comment:
How about:
This step-by-step guide will aid you create a new Konstellation version for your project, which will help you understand all the main KRE concepts and test KRE functions.

Also, I think it would be nice if we put this description under the step by step title to aid eye scheming